### DISCUSSION TITLE
Layer problems

### DISCUSSION THREAD

**Discussion by Rosinnnn** [keiwando/toonsquid#579] [Q&A]
Everytime I draw something on a layer, I’ll eventually draw on the another layer it generates automatically, and I can’t even merge these two layers.
And the new thing I drew on the same drawing layer did not move with the keyframe I set.

Are there any solution for these problem?

https://github.com/keiwando/toonsquid/assets/169423027/e4eb478d-c016-447c-88e2-2aa780658ae1

**Comment by budaloops** [comment]
It looks like the flower is a pixel layer and when you draw on it you are creating a vector layer. These two can’t be merged unless you rasterize your vector drawing layer first. 

You can simply group the flower and the drawn lines on top of it into a group and move it around as a group.

can’t remember the ins and outs of ToonSquid’s interface but that’s the general idea.

**Comment by FoxeyeStudios** [comment]
Ok, the problem here is you are using a vector tool to try and draw on a raster layer. You need to make sure you’re using the tools relevant to the current layer you’re using. Press the brunch icon again to show the types of brushes you have. You need to make sure you’re not using vector brushes unless that’s what you’re intending. You can always merge the layers or rasterize the vector layer too, they’re just extra steps.
